### ISSUE TITLE
 ⚒️ Design/build navigation

### ISSUE THREAD

**Issue by tatianamac** [tatianamac/selfdefined#183]
## Current Behaviour

The site has flat navigation right now, which isn't as accessible and as clear as it could be.

## Desired Behaviour

Clearer navigation that could include:

- [ ] Search bar (#135 )
- [ ] Perhaps categorisations? Show list of terms by flags? 
- [ ] Toggle language (#164 )
- [ ] About page (#178 )

Of course building the navigation requires building all those other pieces, but I think we can consider them a block to this. I'd love for us to imagine the design of this while those pieces are coming into place.

**Comment by tbrasington** [comment]
I would like to help with this. Is there an approach for the design tasks?

**Comment by tatianamac** [comment]
Not yet! I'm not sure what you mean by design approach. Could you elaborate? I'm happy to answer any specific questions.

Also I always invite collaborators to our [Slack channel](https://join.slack.com/t/selfdefined/shared_invite/zt-en1u6xn8-CumcehUfprC0GH5lamb1bg) to discuss with other contributors.

**Comment by tbrasington** [comment]
I was thinking whether ideas where worked on in a tool like Figma, with the work shared for feedback? or building something and doing a PR?

Thank you for the invite :)

**Comment by dasilvacontin** [comment]
It looks like @tbrasington has been working on over this at https://www.figma.com/file/yRxihnosfvOWrLng7KkfF0/Design-System?node-id=101%3A2834, with progress/ideas done on:
- Search bar for the homepage
- Categories and Flags filter
- Language toggle in the footer
- And About page

**Comment by tatianamac** [comment]
Correct, @dasilvacontin! @tbrasington Are you still able and willing to finish this?

**Comment by tbrasington** [comment]
Yep! I have free time again.
